flang/test/Lower/OpenMP/FIR/delayed-privatization-firstprivate.f90

```diff
@@ -0,0 +1,29 @@
+! Test delayed privatization for the `private` clause.
+
+! RUN: bbc -emit-fir -hlfir=false -fopenmp --openmp-enable-delayed-privatization -o - %s 2>&1 | FileCheck %s
+
+subroutine delayed_privatization_firstprivate
+  implicit none
+  integer :: var1
+
+!$OMP PARALLEL FIRSTPRIVATE(var1)
```

Comment:
nit: I'd use lowercase here, as in the last test. My eyes are used to seeing these directives in lowercase :)